packages/ERTP/src/types-module.d.ts

```diff
@@ -0,0 +1,445 @@
+/**
+ * @file Generated from types.js, then find-replace to export all
+ *
+ * This file exists because types.js is ambient and some modules need the types
+ * to be explictly imported. Changing types.js to a module breaks too much.
```

Comment:
Do you have a tool that automatically converted, or was in done manually?

What's the plan for maintenance?  If we will have to manually keep them in sync, that should be mentioned here and there, even if you plan to keep up with it yourself.

Reply:
It was `yarn lint:types` with some find replace. This file should say exactly how to reproduce it 👍